presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -1582,4 +1635,110 @@ public static OperatorHashCollisionsStats merge(OperatorHashCollisionsStats firs
                     first.weightedExpectedHashCollisions + second.weightedExpectedHashCollisions);
         }
     }
+
+    private static class WindowNodeStats
+    {
+        private final List<Long> indexSize;
+        private final List<Long> partitionRowsPerIndex;
+        private final List<Long> partitionRowsPerDriver;
+        private final int activeDrivers;
+        private final int totalDrivers;
+
+        public static WindowNodeStats create(WindowInfo info)
+        {
+            ImmutableList.Builder<Long> indexSize = ImmutableList.builder();
+            ImmutableList.Builder<Long> partitionRowsPerIndex = ImmutableList.builder();
+            info.getWindowInfos().stream()
+                    .map(SingleDriverWindowInfo::getIndexInfo)
+                    .flatMap(Collection::stream)
+                    .forEach(indexInfo -> {
+                        indexSize.add(indexInfo.getSizeInBytes());
+                        partitionRowsPerIndex.add(indexInfo.getTotalRowCountInPartitions());
+                    });
+
+            ImmutableList.Builder<Long> partitionRowsPerDriver = ImmutableList.builder();
+            int activeDrivers = 0;
+            int totalDrivers = 0;
+
+            for (SingleDriverWindowInfo singleDriverWindowInfo : info.getWindowInfos()) {
+                long partitionRowsCount = singleDriverWindowInfo.getIndexInfo().stream()
+                        .mapToLong(IndexInfo::getTotalRowCountInPartitions)
+                        .sum();
+                partitionRowsPerDriver.add(partitionRowsCount);
+                totalDrivers++;
+                if (partitionRowsCount > 0) {
+                    activeDrivers++;
```

Comment:
This is the number of drivers which participated in executing a window function. It has nothing to do with an `Explain Analyze`, nor a `Limit` operator.
The `Active drivers / total drivers` is simply a metric of the parallelism.